charts/mirror/templates/configmap.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "mirror.fullname" . }}
+data:
+  default.conf: |
+    server {
+      listen       80;
+      server_name  localhost;
```

Comment:
is this right?

Reply:
technically yes, it works because it's the default and only server configuration, we use it everywhere in our infra